django_project/lesson/models/section.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
+"""Section model definitions for lesson apps.
+
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from unidecode import unidecode
+from core.settings.contrib import STOP_WORDS
+from django.utils.text import slugify
+import logging
+logger = logging.getLogger(__name__)
+
+
+class Section(models.Model):
+    """Section lesson model."""
```

Comment:
A section is a grouping for one or more lessons on a common theme e.g. digitising. Sections can be ordered so that the learner goes through the sections in the correct sequence.